client/my-sites/email/titan-set-up-mailbox/titan-set-up-mailbox-form.tsx

```diff
@@ -91,10 +86,17 @@ const useHandleCompleteSetup = (
 		try {
 			await mutateAsync( mailbox.getAsFlatObject() as TitanMailboxFields );
 
-			goToThankYouPage(
-				selectedSite?.slug as string,
-				selectedDomainName,
-				mailbox.formFields.mailbox.value
+			// The provision of new user (mailbox) is done through an async process, which means
+			// the new mailbox won't be immediately available. To account for this, we redirect
+			// users back to the Email Management page with a 5 seconds delay, to prevent
+			// users from seeing the "Set up mailbox" CTA again. There's a chance that the
+			// new mailbox is still unavailable after 5 seconds, but that is an edge case that we will
+			// cope with for the time being.
+			await new Promise( ( resolve ) => setTimeout( resolve, 5000 ) );
+
+			page( emailManagementEdit( selectedSite?.slug as string, selectedDomainName ) );
```

Comment:
```suggestion
			page( getEmailManagementPath( selectedSite?.slug, selectedDomainName ) );
```
Let's use `getEmailManagementPath` from `client/my-sites/email/paths.ts` here! That's the primary function for retrieving a path to the email management page.

I would almost argue that we should remove `emailManagementEdit` in `packages/domains-table/src/utils/paths.ts`, since it duplicates logic from `getEmailManagementPath`, but I'm unsure if there are additional considerations to this, given that `emailManagementEdit` lives in a `packages` folder and `getEmailManagementPath` lives in the `client` folder. Maybe @chad1008 has some insight there..?

Reply:
re: removing `emailManagementEdit`, the main concern I'd have is the obvious one of making sure it's not being used anywhere else in the codebase, but it's entirely possible there are other concerns I'm not considering 🤔